README.md

```diff
@@ -102,11 +109,37 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 
 #### GitHub permission scopes
 
+##### Repository Permissions
+
+|**Permission scope**|**Why the app needs it**|
+|---|---|
+|**Read** access to contents & metadata |**Contents** (aka code) and **metadata** are needed to sync development information to Jira. <br><br> **Medadata:** All GitHub apps have read-only metadata permission set by default. This is a [mandatory requirement by GitHub](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) and is needed to provide access to a collection of read-only endpoints with metadata for various resources. These endpoints do not leak sensitive private repository information. Read-only metadata permissions are used for the following webhook: <br> - repository (all events excluding `deleted`) <br><br> **Contents:** Read-only content permissions are used for the following webhooks: <br> - commit comment <br> - create <br> - delete <br> - push <br> - workflow run|
+|**Read** and **write** access to issues and pull requests|**Issues** and **pull requests** are used by the GitHub for Jira app to power Smart Commit actions and unfurl Jira URLs. This allows us to create links to Jira issues from GitHub pull request comments or issue comments when you include an issue key in the body e.g. [ABC-123] will link to Jira issue ABC-123 <br><br> **Issues:** Read and write issue permissions are used for the following webhooks: <br> - issue comment <br> - issues <br><br> **Pull requests:** Read and write pull request permissions are used for the following webhooks: <br> - pull request <br> - pull request review|
+|**Read** and **write** access to actions and deployments|If you want to see build and deployment information in Jira, the app will need read and write permissions for deployments. In order for the GitHub for Jira app to send the correct status (pending, in progress, successful etc) of a deployment to Jira we require the `state` property. This property is only available on the `deployment_status` event which occurs when a deployment is created. The [GitHub API](https://docs.github.com/en/rest/reference/repos#create-a-deployment-status) requires that GitHub apps have read and write access to listen to deployment creation events. <br><br> **Deployments:** Read and write deployment permissions are used for the following webhooks: <br> - deployment status|
```

Comment:
The app is not calling the API to create (POST) deployment statuses (it shouldn't be, at least!). It should be calling [this API](https://docs.github.com/en/rest/reference/repos#list-deployment-statuses) to list deployment status. The docs say that this only requires "pull access". Is this "pull access" the missing piece perhaps? Can we give "pull" access without giving "write" access to deployments?

Reply:
No, it's not making a POST request but `deployment_status` is fired when a deployment is created https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#deployment_status and, according to the GitHub documentation, GitHub apps require read and write access https://docs.github.com/en/rest/reference/repos#create-a-deployment-status

Reply:
The deployment is created by a GitHub action or some external tool like Jenkins. _That_ app needs write access to create a deployment, which causes the `deployment_status` webhook, which gets sent to the GitHub app. 

I understand the docs that write access is required for _creating a deployment_, which makes sense. But the GitHub app is not creating a deployment. It's consuming `deployment_status` events.

Reply:
Looking into the docs of the `deployment_status` event, I see this:

![image](https://user-images.githubusercontent.com/1363578/144353249-21261c46-7562-4c30-ade6-fee9c8943aef.png)

Sadly, this only says `deployments` permission on not whether it's read or write.